reo/migrations/0107_merge_20210420_1418.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.1.8 on 2021-04-20 14:18
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
We could probably squash all of these migrations into one as well, or make one migration for each new model.

Reply:
Ok, I can do that. Do you know if that will require undeploying and redeploying the branch on staging to avoid db errors?

Reply:
I squashed the migrations when I split the GHPGHX models into Inputs and Outputs